borgmatic/borg/create.py

```diff
@@ -150,11 +151,16 @@ def create_archive(
         + (('--umask', str(umask)) if umask else ())
         + (('--lock-wait', str(lock_wait)) if lock_wait else ())
         + (('--list', '--filter', 'AME',) if logger.isEnabledFor(logging.INFO) else ())
-        + (( '--info',) if logger.getEffectiveLevel() == logging.INFO else ())
-        + (('--stats',) if not dry_run and logger.isEnabledFor(logging.INFO) else ())
+        + (('--stats',) if not dry_run else ('--dry-run',))
         + (('--debug', '--show-rc') if logger.isEnabledFor(logging.DEBUG) else ())
-        + (('--dry-run',) if dry_run else ())
     )
 
     logger.debug(' '.join(full_command))
-    subprocess.check_call(full_command)
+    output = subprocess.check_output(full_command, stderr=subprocess.STDOUT, universal_newlines=True)
+    if logger.isEnabledFor(logging.INFO):
+        logger.debug(output)
```

Comment:
This is tricky. Well first of, I didn't put too much thought into it :flushed: .
But on second thought, I can't really detect what the `borg` level is, unless i use `--log-json`. 

As said earlier, I'll gladly port it all to the json-output, but upon giving borg create `--json` and `--log-json` the `--json` takes precedence and does not give me a loglevel.

Also although I agree, that it would be nicer to use the `create --json`, we would have to re-build the human readable output based on the json. Otherwise it'd not really be backwards compatible in case anybody is parsing the stdout-output for similar purposes.